node/silkworm/db/util.hpp

```diff
@@ -154,6 +154,10 @@ Bytes storage_prefix(ByteView address, uint64_t incarnation);
 // Erigon CanonicalHeadersKey / ReceiptsKey
 Bytes block_key(uint64_t block_number);
 
+// Erigon TxIndex value
+Bytes encode_lookup(uint64_t lookup);
```

Comment:
they encode lookups, lookups are generic numbers just "compacted". i was planning on doing some experimentation with new lookups tables and i just put it there because in the future it is likely we will add more of those anyway.